LPS-82545 WebSphere 8.5.5.x doesn't provide the javaext dir, so portlet.jar has to be put in jre/lib/ext

dantewang opened the pull request.

jkappler committed:
LPS-82444 Use UUID from the service context

jkappler committed:
LPS-82444 Listen to changes in layout page template collection and entries

jkappler committed:
LPS-82444 Needed for changesets

jkappler committed:
LPS-82444 Fix typo

jkappler committed:
LPS-82444 Make it only visible in staging or when staging is not active

jkappler committed:
LPS-82444 For staging, we have to take into account all the fields when creating the new entry, so adapt accordingly

jkappler committed:
LPS-82444 Staging Handlers for asset display pages

jkappler committed:
LPS-82444 Export/Import of display pages for journal articles and for layout page template entries

jkappler committed:
LPS-82444 Add modified date when updating entries

Commit:
LPS-82444 Consider last publish dates

Commit:
LPS-82444 The pattern is to make the checkbox visible but disable as it's not optional to publish or not

dantewang committed:
LPS-82545 WebSphere 8.5.5.x doesn't provide the javaext dir, so portlet.jar has to be put in jre/lib/ext

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

ealonso committed:
LPS-82444 SF

jkappler committed:
LPS-82572 Auto

jkappler committed:
LPS-82572 Use new method

jkappler committed:
LPS-82444 We need to update the display page associated, mostly for listening for changes for staging

jkappler committed:
LPS-82444 Baseline

rafaprax committed:
LPS-82648 Delete structures after delete the form's entries and use the right delete method

mtambara committed:
LPS-82333 Make sure to always return a full locale, not just the language

tinatian commented:
@alee8888 Could you please test this on webshpere 8? I did the same thing manually, and it works. But I have problem building it with scripts, could you please verfiy?

Thanks.
Tina.

Ithildir committed:
LPS-82857 Find the oldest commit in a branch

Ithildir committed:
LPS-67653 Make methods private to reduce API surface

Ithildir committed:
LPS-82857 Export package

Ithildir committed:
LPS-82857 Add packageinfo

Ithildir committed:
LPS-82857 Enforce semantic versioning

Ithildir committed:
LPS-67653 Move unexported package to "internal"

Ithildir committed:
LPS-82857 Semver gradle-plugins-change-log-builder

Ithildir committed:
LPS-67653 Update changelog

Ithildir committed:
LPS-82857 Update changelog

Ithildir committed:
LPS-82857 Use latest gradle-plugins-change-log-builder

Preston-Crary committed:
LPS-82860 Do not create the layout just to delete it.

brianchiu committed:
LPS-81423 Location assertion not needed, just wait for page reload

lesliewong92 committed:
LRQA-41133 Use repository name instead of repository directory path

michaelhashimoto committed:
LRQA-41877 Only deploy a new source format jar when the module is available

jkappler committed:
LPS-82444 Auto SF

brianchandotcom committed:
LRQA-41133 prep next

shuyangzhou committed:
LPS-82333 SF

Commit:
LPS-82850 Disable the failing tests for the moment

They only fail when ran from the suite but work when run separately, so
the feature seem to work but there exists some interaction in the test
suite.

brianchandotcom committed:
LPS-82444 Hugo SF

brianchandotcom committed:
LPS-82444 SF

brianchandotcom committed:
LPS-82828 prep next

brianchandotcom committed:
LPS-82857 prep next

brianchandotcom committed:
Record reference to liferay-portal/master-private with git-commit-portal-private as 2209fccc19cdc1d94866e8c04f72405f0f4a69d6

petershin committed:
LPS-0 Update 'com.liferay.portal.kernel.util' version

dantewang commented:
see https://github.com/tinatian/liferay-portal/pull/912
